### ISSUE TITLE
Many 1.19 docker images contain the wrong openresty version

### ISSUE THREAD

**Issue by pgibson1-godaddy** [openresty/docker-openresty#211]
It seems like a bunch of the `1.19.9.1-13` docker images actually contain the wrong version of openresty (or at least report that they do).

For example:

```
$ docker run --rm -it openresty/openresty:1.19.9.1-13-alpine-fat openresty -V | head -n 1
nginx version: openresty/1.21.4.1
```

```
$ docker run --rm -it openresty/openresty:1.19.9.1-13-buster-fat openresty -V | head -n 1
nginx version: openresty/1.21.4.1
```

```
$ docker run --rm -it openresty/openresty:1.19.9.1-13-bullseye-fat openresty -V | head -n 1
nginx version: openresty/1.21.4.1
```

There are probably others affected as well.

Separately, OpenSSL `1.1.1q` is released as well so it would be nice to get builds based on that.

**Comment by neomantra** [comment]
Thanks for bringing these to my attention -- seems only be the fat images?

I opened #212 for the OpenSSL.

**Comment by pgibson1-godaddy** [comment]
Yes looks like you are right, and thank you!

**Comment by neomantra** [comment]
I'm working on a fix for this -- it only affects "fat" images with the tag `1.19.9.1-13`, which emerged parallel to the 1.21 series.  The issue was with how the CI was set up.

**Comment by neomantra** [comment]
I am still working on this, but I have removed tags `1.19.9.1-13-*-fat` from Docker Hub.

**Comment by neomantra** [comment]
I’ve released tag `1.19.9.1-14` which resolves this.  I made the base images more explicit and also had to explicitly specify the `openresty-opm` package versions.  The `resty` utility was missing in the Debian flavors, so I added that too. 
Thanks again for reporting.

**Comment by neomantra** [comment]
Released in `1.21.4.1-3` and `1.19.1.1-14`.
